src/EditorFeatures/CSharpTest/Diagnostics/UseImplicitOrExplicitType/UseImplicitTypeTests.cs

```diff
@@ -1730,6 +1769,21 @@ static void M()
 options: ImplicitTypeEverywhere());
         }
 
+        [WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsUseExplicitType)]
+        [WorkItem(23893, "https://github.com/dotnet/roslyn/issues/23893")]
+        public async Task SuggestVarOnDeclarationExpressionSyntaxWithIntrinsicType()
```

Comment:
💭 Name seems to contradict test, but it may not be the only test with this pattern. Maybe no change here, but make sure the current name was intended?